assets/src/features/shared/components/PlainLink.tsx

```diff
@@ -24,15 +24,22 @@ const PlainLink: React.FC<PlainLinkProps> = ({ href, className, name, children,
   // external links
   if (href?.startsWith("http://") || href?.startsWith("https://")) {
     return (
-      <a target="_blank" rel="noreferrer" href={href} className={clsx(className)} aria-label={name} onClick={onClick}>
+      <a
+        target="_blank"
+        rel="noreferrer"
+        href={disabled || !href ? "" : href}
```

Comment:
1. `!href ? "" : href` can be reduced to `href ?? ""`
2. Since the pattern `if disabled then <noop> else <do_something>` is repeated 4 times in this component, maybe it could be wrapped in a function?

Reply:
1. but it is `(disabled || !href) ? "" : href`

Reply:
Oh, all right then